lib/rate-limiter.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const _ = require('lodash');
+
+let rateTable = {};
```

Comment:
Thoughts about having this function return the mapping of routes and the `execute` function take the mapping as an argument to make it a bit more functional and stateless. It'll aid in testing and allow you to avoid these types of globals.